client/src/pages/admin/CreateCategory.test.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import axios from "axios";
+import { render, waitFor, screen, fireEvent } from '@testing-library/react';
+import { MemoryRouter, Routes, Route } from 'react-router-dom';
+import CreateCategory from './CreateCategory';
+import '@testing-library/jest-dom/extend-expect';
+
+//Mock components
+jest.mock('axios');
+jest.mock('../../components/AdminMenu', () => () => <div>Admin Menu Mock</div>);
+jest.mock('./../../components/Layout', () => ({ children }) => <div><h1>Dashboard Mock</h1><div>{ children }</div></div>);
+jest.mock('./../../components/Form/CategoryForm', () => ({ children }) => <div><h1>Category Form Mock</h1><div>{ children }</div></div>);
+
+
+describe("Create Category Component", () => {
+
+    beforeEach(() => {
+        jest.clearAllMocks();
+    });
+
+
+    describe("When the component is rendered", () => {
+        
+        test("The different components are rendered with successfull fetching of categories", async () => {
```

Comment:
If this test is not supposed to pass, should we use `test.failing`?

Reply:
Or is it supposed to pass, but the comment for assert is incorrect?

Reply:
Assertion corrected! Tysm!